bawan-shna-abdulrahman/src/Components/RenderMovie.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+
+import { Modal, Button, Card, CardTitle, CardText } from "react-bootstrap";
+
+export default function RenderMovie(props) {
+  const [show, setShow] = useState(false);
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
```

Comment:
you can instead of that use one function and use the previous state to set the new state